src/game/bundles/entity.rs

```diff
@@ -194,8 +194,8 @@ impl MonsterBundle {
 pub struct DroppedItemBundle {
     pub item: Option<DroppedItem>,
     pub position: Position,
-    pub owner: Owner,
-    pub expire_time: ExpireTime,
+    pub owner: Option<Owner>,
```

Comment:
Also unfortunately this doesn't work as bevy will insert a component of type Option<Owner>, rather than optionally insert an Owner.

What we have to do here is remove owner from the bundle (because bundles do not support optional components) and insert it separately.

```
#[derive(Bundle)]
pub struct DroppedItemBundle {
    pub item: Option<DroppedItem>,
    pub position: Position,
    pub entity_expire_time: EntityExpireTime,
}

impl DroppedItemBundle {
    pub fn spawn(
        commands: &mut Commands,
        client_entity_list: &mut ClientEntityList,
        item: DroppedItem,
        position: &Position,
        owner_entity: Option<Entity>,
        server_time: &ServerTime,
    ) -> Option<Entity> {
    
    ....
    
        entity_commands.insert_bundle(DroppedItemBundle {
            item: Some(item),
            position: drop_position.clone(),
            expire_time: ExpireTime::new(server_time.now + DROPPED_ITEM_EXPIRE_TIME),
        });

        if let Some(owner_entity) = owner_entity {
            entity_commands.insert(Owner::new(owner_entity));
        }

```